README.md

```diff
@@ -1,14 +1,12 @@
 # CircleCI Server Setup
 
-This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform. For a full step by step guide to installing CircleCI Server with Terraform, see our [installation guide](https://circleci.com/docs/2.0/circleci-install-doc-v2-17.pdf#section=administration).
+This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform.
 
 **Note: This is only meant to be used for the initial setup of CircleCI Server and is not meant to be used for the ongoing maintenance of the CircleCI Server.**
 
 **Note: Master is the only supported branch. All other branches of this repo should not be considered stable, and is to be used at your own risk.**
 
-# Getting Started
-
-## Pre Reqs
+## Documentation
```

Comment:
Might be worth pointing people to the Prerequisites and Planning doc as well the install guide? https://circleci.com/docs/2.0/aws-prereq/#section=server-administration. This details needing terraform, and all the other bits of info they are going to need to have to hand.

Reply:
Done